__tests__/CustomNotification.test.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import shallowToJSON from 'enzyme-to-json';
+
+import CustomNotification from '../src/Components/CustomNotification/CustomNotification.Component';
+
+describe('CustomNotification component', () => {
+  it('should have all the CustomNotification content', () => {
+    const notification = shallow(<CustomNotification type="success" open={true} message="A test success notification" />);
+    const tree = shallowToJSON(notification);
+    expect(tree).toMatchSnapshot();
```

Comment:
I don't think we'll be needing any of these snapshot tests now or in future.

Reply:
I have made the changes from the snapshots.